### PR TITLE
unions no longer make firearms

### PR DESCRIPTION
**Pull request by elgeonmb**

### CODE REVIEW

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/energy/gun
 	name = "energy gun"
-	desc = "Another bestseller of Lawson Arms and the FTU, the LAEP90 Perun is a versatile energy based sidearm, capable of switching between low and high capacity projectile settings. In other words: Stun or Kill."
+	desc = "Another bestseller of Lawson Arms and"+TSC_HEPH+", the LAEP90 Perun is a versatile energy based sidearm, capable of switching between low and high capacity projectile settings. In other words: Stun or Kill."
```
Comment (Neerti) [inline code/modules/projectiles/guns/energy/nuclear.dm]: You'll need a space between the ``and"`` and ``+TSC_HEPH``, or else it'll say ``Another bestsellar of Lawson Arms andHephethsus`` or however it's actually spelled (yay for that constant)

Comment (Neerti) [reply code/modules/projectiles/guns/energy/nuclear.dm]: Apparently you already got it so wew.

**Comment by Arokha** [comment]
Travis is being a dick currently and that build timed out.

**Comment by Anewbe** [comment]
Tested?

**Comment by Atermonera** [comment]
TSC_HEPH is #define'd as "Hephaestus", so that's just string addition, which works as you'd expect

**Pull request merged**